### PR TITLE
Custom relationship mapper keys

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

**Comment by 3lvis** [comment]
@RamonGilabert This broke :(

File: Source/NSManagedObject+HYPPropertyMapper.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import CoreData;
 
 static NSString * const HYPPropertyMapperCustomRemoteKey = @"mapper.remote.key";
```
Comment (3lvis) [inline Source/NSManagedObject+HYPPropertyMapper.h]: I think we can reuse the same key, we don't have to specific a different one. The difference is that one is in an attribute while the other on a relationship.

**Comment by RamonGilabert** [comment]
What happened @3lvis? :o

**Comment by RamonGilabert** [comment]
Nice @3lvis! :dancer:

**Comment by 3lvis** [comment]
@RamonGilabert should this be enough? https://github.com/hyperoslo/Sync/pull/36

I think so, maybe we should close this PR

**Pull request closed without merging**